orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/helpers/ClientSubjectHelper.java

```diff
@@ -95,17 +95,27 @@ public static String calculatePairwiseIdentifier(String subjectID, URI uri, byte
 
     public static String calculatePairwiseIdentifier(
             String subjectID, String sectorHost, byte[] salt) {
+        var md = getMessageDigest(sectorHost, subjectID);
+        byte[] bytes = md.digest(salt);
+        var sb = Base64Url.encode(bytes);
+        return PAIRWISE_PREFIX + sb;
+    }
+
+    public static String calculateWalletSubjectIdentifier(String sectorID, String commonSubjectID) {
+        var md = getMessageDigest(sectorID, commonSubjectID);
+        byte[] bytes = md.digest();
+        var sb = Base64Url.encode(bytes);
+        return PAIRWISE_PREFIX + sb;
```

Comment:
Asked for confirmation we're using the same pairwise prefix: https://gds.slack.com/archives/C066NQKE95G/p1709043871964709